ch12/exp_loss.md

```diff
@@ -0,0 +1,52 @@
+# Expected Loss
+Let us return to our discussion of the constant model with the tip percentage dataset in the previous chapters. Recall that we wanted to predict a single tip percentage for all tables using a simple random sample of tips for a particular waiter. Since we only had access to data from this particular sample, we found the model that empirically minimized the loss for data in the sample with the mean absolute, mean squared, and Huber loss functions. If we wanted to generalize from the tips sample to the entire population, however, we would need another loss function that does not specifically depend on which data points appear in our sample. The *expected loss* is an example of such a loss function that allows us to analyze the relationship between our sample and the population.
+
+## Definition
+We define the random variable $ X $ as the percentage tipped for a particular table in the population. As before, we use the constant $ \theta $ to represent our model for the universal tip percentage. Using the quadratic loss as an example, we obtain the following loss function:
+
+\\[ f(\theta) =  \mathbb{E}[(X - \theta)^2] \\]
+
+## Minimizing the Expected Squared Loss
+Note that the expected squared loss is a function of $ \theta $ since $ \theta $ is a model parameter that we get to choose. After some simplification with the linearity of expectation, we can use calculus to find the value of $ \theta $ that minimizes the expected squared loss.
+
+\\[ \begin{aligned}
+f(\theta) &=  \mathbb{E}[(X - \theta)^2] \\\\
+&= \mathbb{E}[X^2 - 2\theta X + \theta^2] \\\\
+&= \mathbb{E}[X^2] - \mathbb{E}[2\theta X] + \mathbb{E}[\theta^2] \\\\
+&= \mathbb{E}[X^2] - 2\theta \mathbb{E}[X] + \theta^2 \\\\
+\end{aligned} \\]
+
+To find the value of $ \theta $ that minimizes this function, we take the derivative of the final expression with respect to $ \theta $.
+
+\\[ \frac{\partial}{\partial \theta}f(\theta) = -2\mathbb{E}[X] + 2\theta \\]
+
+Now, we can find the value of $ \theta $ where the derivative is zero.
+
+\\[ \begin{aligned}
+-2\mathbb{E}[X] + 2\theta &= 0 \\\\
+2\theta &= 2\mathbb{E}[X] \\\\
+\theta &= \mathbb{E}[X] \\\\
+\end{aligned} \\]
+
+Thus, $\theta = \mathbb{E}[X]$ minimizes the expected squared loss. If we plug in this value back into the original loss function, we get:
+
+\\[ f(\mathbb{E}[X]) = \mathbb{E}[(X - \mathbb{E}[X])^2] \\]
+
+Notice that this is simply the definition of $Var(X)$, which means that $Var(X)$ is the minimized error of the expected squared loss.
```

Comment:
I think it'll be good to explain the intuition a bit more here since it's a neat result. For example, if Var(X) = 0 and there is no spread in the population, theta = E[X] perfectly predicts every point and the error is zero. If Var(X) is high (there is lots of spread), many points will be far away from E[X] and the error will be high.

Reply:
Done